src/test/java/com/epam/tc/gdi/hw1/tests/steps/Preconditions.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.gdi.hw1.tests.steps;
+
+import com.epam.tc.gdi.hw1.site.entities.Users;
+import com.epam.tc.gdi.hw1.site.pages.components.complex.Header;
+
+public class Preconditions {
+
+    public static void loginIfNot() {
```

Comment:
It's better to use the same notation: shouldBeLoggedIn

Reply:
Renamed